en/docs/deploy/get-started/customize-the-management-console.md

```diff
@@ -109,7 +102,7 @@ framework instead of the default `global.css`stylesheet.
 ### Change the product specific styles and images
 
 The styles and images unique to your product are located in the
-`org.wso2.<product-name>. styles_<version_number>.jar`.
+`org.wso2.<product-name>. styles_<version_number>.jar` file.
```

Comment:
```suggestion
`org.wso2.identity.styles_<version-number>.jar` file.
```

Reply:
saw that throughout this doc the placeholder `<product-name>` has been used. let's change it to identity since we are only focusing on the identity server as the product